Simplenote/CoreDataValidator.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CoreDataValidator.swift
+//  Simplenote
+//
+//  Created by Charlie Scheer on 5/30/24.
+//  Copyright © 2024 Simperium. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class CoreDataValidator {
+    func validateStorage(withModelURL modelURL: URL, storageURL: URL) throws {
+        guard let mom = NSManagedObjectModel(contentsOf: modelURL) else {
+            fatalError("Could not load Managed Object Model at path: \(storageURL.path)")
+        }
+
+        try loadPersistentStorage(with: mom, at: storageURL)
+    }
+
+    func loadPersistentStorage(with mom: NSManagedObjectModel, at storageURL: URL) throws {
```

Comment:
This API doesn't load, but instead, verifies that it can be loaded. How about something like this?

```
    func validateStorage(model: NSManagedObjectModel, storageURL: URL) throws 
```